### PR TITLE
cspki: add pubkey initialization check

### PR DESCRIPTION
**Pull request by jvehent**

### CODE REVIEW

File: signer/contentsignaturepki/x509.go [hunk 1]
```diff
@@ -37,6 +37,10 @@ func (s *ContentSigner) findAndSetEE(conf signer.Configuration) (err error) {
 		err = errors.Wrapf(err, "found suitable end-entity labeled %q in database but not in hsm", s.eeLabel)
 		return
 	}
+	if len(s.PublicKey) < 50 {
```
Comment (g-k) [inline signer/contentsignaturepki/x509.go]: Should we check this in `GetKeys` / will we ever use a public key < 50 bytes?

File: signer/contentsignaturepki/x509.go [hunk 2]
```diff
@@ -37,6 +37,10 @@ func (s *ContentSigner) findAndSetEE(conf signer.Configuration) (err error) {
...
+		err = errors.Errorf("malformed public key in key label %q for signer %q: %q", s.eeLabel, s.ID, s.PublicKey)
```
Comment (g-k) [inline signer/contentsignaturepki/x509.go]: Worth mentioning how the keys is malformed in the error message.

File: signer/contentsignaturepki/x509.go (hunk 1 above)
Comment (jvehent) [reply signer/contentsignaturepki/x509.go]: makes sense, moved it there

File: signer/contentsignaturepki/x509.go (hunk 2 above)
Comment (jvehent) [reply signer/contentsignaturepki/x509.go]: changed the message to mention it, but at that point we don't know _why_ it's malformed

Comment (g-k) [reply signer/contentsignaturepki/x509.go]: well why we're considering it malformed

**Pull request merged**